ee/tables/execparsers/socketfilterfw/parser.go

```diff
@@ -0,0 +1,84 @@
+package socketfilterfw
+
+import (
+	"bufio"
+	"io"
+	"regexp"
+	"strings"
+)
+
+var lineRegex = regexp.MustCompile("(state|block|built-in|downloaded|stealth|log mode|log option)(?:.*\\s)([0-9a-z]+)")
+
+// socketfilterfw returns lines for each `get` argument supplied.
+// The output data is in the same order as the supplied arguments.
+//
+// Each line describes a part of the feature and what state it's in.
+// These are not very well-formed, so I'm doing some regex magic to
+// know which option the current line is, and then sanitize the state.
+func socketfilterfwParse(reader io.Reader) (any, error) {
+	results := make([]map[string]string, 0)
+	row := make(map[string]string)
+
+	scanner := bufio.NewScanner(reader)
+	for scanner.Scan() {
+		line := strings.ToLower(scanner.Text())
+		matches := lineRegex.FindStringSubmatch(line)
+		if len(matches) != 3 {
+			continue
+		}
+
+		var key string
+		switch matches[1] {
+		case "state":
+			key = "global_state_enabled"
+		case "block":
+			key = "block_all_enabled"
+		case "built-in":
+			key = "allow_built-in_signed_enabled"
+		case "downloaded":
+			key = "allow_downloaded_signed_enabled"
+		case "stealth":
+			key = "stealth_enabled"
+		case "log mode":
+			key = "logging_enabled"
+		case "log option":
+			key = "logging_option"
+		default:
+			continue
+		}
+
+		// Don't allow overwrites.
+		_, ok := row[key]
```

Comment:
When would this occur?

Reply:
I had placed this in for the malformed test data as the `state` string for global state is matched in the app states. Now that data shouldn't be ever received, so this may be overkill.